README.md

```diff
@@ -13,7 +13,7 @@ Other useful technical documentation:
 - [micro:bit Tech Site DAPLink page](https://tech.microbit.org/software/daplink-interface/)
 - [micro:bit I2C Specification](https://github.com/microbit-foundation/spec-i2c-protocol)
 - [micro:bit Tech Site pin and signals information](https://tech.microbit.org/hardware/schematic/)
-- micro:bit V2 schematic
+- [micro:bit V2 schematic](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/MicroBit_V2.0.0_S_schematic.PDF)
```

Comment:
Thanks for adding this as well!
I think it's fine to just point to the repo here, links to specific files can break in the future and this way they enter through the README.
```suggestion
- [micro:bit V2 schematic](https://github.com/microbit-foundation/microbit-v2-hardware/)
```